debian/rules

```diff
@@ -11,14 +11,21 @@
 export VERBOSE=1
 
 %:
-	dh $@ --parallel --with autotools_dev,autoreconf,bash_completion,python2
+	dh $@ --parallel --with autotools_dev,autoreconf,bash_completion,python3
 
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-python-libs --enable-rdm-tests
+	dh_auto_configure -- --enable-python-libs --enable-rdm-tests CXXFLAGS='-Wno-error=deprecated-declarations -Wno-error=unused-parameter' pythondir='/usr/lib/python3/dist-packages'
 
 override_dh_installinit:
 	dh_installinit -p ola --name=olad
 	dh_installinit -p ola-rdm-tests --name=rdm_test_server
 
 override_dh_makeshlibs:
 	dh_makeshlibs -V
+
+override_dh_clean:
+	find . -type d -name '__pycache__' -print0 | xargs -0 rm -rf
```

Comment:
I thought there might be a better clean tool and it seems it's this:
https://manpages.ubuntu.com/manpages/bionic/man1/py3clean.1.html

Reply:
Maybe also check what other packages use?

Reply:
I got this from downstream: https://salsa.debian.org/wouter/ola/-/blob/4a0ec01039b06f8ff2d350520a1d031b86e7920b/debian/rules#L27-29

Not many results (although this is not a definitive search) of using `py3clean` with an override: https://www.google.com/search?q=%22override_dh_clean%22+py3clean

It seems that `dh_python3` [may run `py3clean` at some point (`prerm-py3clean` step)](https://salsa.debian.org/python-team/tools/dh-python/-/blob/master/autoscripts/prerm-py3clean) but given that this was deemed necessary downstream I sort of trust that it is deleting something that isn't getting removed otherwise? I suppose this should be easy to test out.

Reply:
OK, I checked all of the `*.deb` files in the bullseye build and there are not any `__pycache__` folders, `*.pyc` files or `*.pyo` files!

Reply:
> @yoe is there some technical reason we're missing why this needs to be included? Maybe because we build and compile files for ola-rdm-tests, but you don't currently clean them correctly as you don't currently install/package ola-rdm-tests?

tbh, I'm not sure why I added that line and not the py3clean one. For sure the cache and pyc and pyo files are required to not be shipped but be generated on Debian systems, so the fact that we remove them before packaging is not something we can get rid of, but how we do it is very much open for discussion...